[2025-09-13T17:55:43.323+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-13T17:55:43.395+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.analyze scheduled__2025-09-12T16:00:00+00:00 [queued]>
[2025-09-13T17:55:43.416+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.analyze scheduled__2025-09-12T16:00:00+00:00 [queued]>
[2025-09-13T17:55:43.417+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-09-13T17:55:43.443+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): analyze> on 2025-09-12 16:00:00+00:00
[2025-09-13T17:55:43.462+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'analyze', 'scheduled__2025-09-12T16:00:00+00:00', '--job-id', '250', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmpa78hwrut']
[2025-09-13T17:55:43.467+0000] {standard_task_runner.py:105} INFO - Job 250: Subtask analyze
[2025-09-13T17:55:43.467+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1258) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-13T17:55:43.469+0000] {standard_task_runner.py:72} INFO - Started process 1259 to run task
[2025-09-13T17:55:43.687+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.analyze scheduled__2025-09-12T16:00:00+00:00 [running]> on host 6eb00eb09733
[2025-09-13T17:55:43.979+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='analyze' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-12T16:00:00+00:00'
[2025-09-13T17:55:43.980+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-13T17:55:44.001+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-13T17:55:44.004+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/analyze.py']
[2025-09-13T17:55:44.025+0000] {subprocess.py:86} INFO - Output:
[2025-09-13T17:55:45.943+0000] {subprocess.py:93} INFO - /opt/***/scripts/analyze.py:157: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
[2025-09-13T17:55:45.944+0000] {subprocess.py:93} INFO -   today = datetime.utcnow().strftime("%Y-%m-%d")
[2025-09-13T17:55:45.972+0000] {subprocess.py:93} INFO - + Text report saved to /opt/***/data/reports/report_2025-09-13.txt
[2025-09-13T17:55:46.117+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-13T17:55:46.236+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-13T17:55:46.237+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=analyze, run_id=scheduled__2025-09-12T16:00:00+00:00, execution_date=20250912T160000, start_date=20250913T175543, end_date=20250913T175546
[2025-09-13T17:55:46.298+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-13T17:55:46.378+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-13T17:55:46.382+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
